devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -33,7 +33,8 @@ metadata {
 		attribute "colorName", "string"
 
 		// Generic fingerprint
-		fingerprint profileId: "0104", deviceId: "010D", inClusters: "0006, 0008, 0300"// deviceId 0x010D is "Extended Color Light"
+		fingerprint profileId: "0104", deviceId: "0102", inClusters: "0006, 0008, 0300", deviceJoinName: "Generic RGBW Light"
```

Comment:
Do we want to give these ocfDeviceType of "oic.d.light"?

Reply:
Hmm.. shouldn't this whole file use that `ocfDeviceType`?

Reply:
I'm just going to leave it as is and we can address this in another PR if necessary